sandpack-react/package.json

```diff
@@ -66,8 +66,6 @@
     "dotenv": "^8.2.0",
     "postcss": "^8.2.6",
     "postcss-cli": "^8.3.1",
-    "react": "16.13.1",
```

Comment:
@alexnm can you any potential risk for this? It causes conflicts with the main application, and probably it will reduce the bundle of Sandpack. As react/react-dom is on `peerDependencies` I cannot see any problem, but let me know your thoughts.